<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateTimeDialog</class>
 <widget class="QDialog" name="DateTimeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="fromlabel">
     <property name="text">
      <string>From:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="fromDateTimeEdit">
     <property name="displayFormat">
      <string>dd.MM.yyyy hh:mm</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="tolabel">
     <property name="text">
      <string>To:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="toDateTimeEdit">
     <property name="displayFormat">
      <string>dd.MM.yyyy hh:mm</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="cancelPushButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okPushButton">
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
